modules/4337/.env.sample

```diff
@@ -30,3 +30,6 @@ DEPLOYMENT_ENTRY_POINT_ADDRESS=""
 TEST_BUNDLER_URL=""
 # The mnemonic used by the bundler for executing user operations
 TEST_BUNDLER_MNEMONIC=""
+
+SOLIDITY_VERSION= # Example: "0.8.19"
+SOLIDITY_SETTINGS= # Example: '{"viaIR":true,"optimizer":{"enabled":true, "details": {"yul": true, "yulDetails": { "optimizerSteps": ""}}}}'
```

Comment:
This example is suboptimal.

1. `yul` flag is not needed since 0.6.0, it's the default
2. `optimizerSteps` is empty. From the docs:
>            // If set to an empty value, only the default clean-up sequence is used and
>           // no optimization steps are applied.
>           "optimizerSteps": "dhfoDgvulfnTUtnIf..."
3. I also think specifying the `details` property might mess up with defaults because the compilation works when I pass it but doesn't when I omit it.

Docs: https://docs.soliditylang.org/en/v0.8.23/using-the-compiler.html#input-description